phpunit.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         bootstrap="tests/bootstrap.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnError="false"
+         stopOnFailure="false"
+         verbose="true"
+>
+  <testsuites>
+    <testsuite name="Twill Test Suites">
+      <directory suffix="Test.php">./tests</directory>
+    </testsuite>
+  </testsuites>
+  <filter>
+    <whitelist processUncoveredFilesFromWhitelist="true">
+      <directory suffix=".php">./src</directory>
+    </whitelist>
+  </filter>
+  <logging>
+    <log type="coverage-html" target="tests/coverage/" lowUpperBound="35" highLowerBound="70"/>
+    <log type="coverage-clover" target="tests/coverage/coverage-clover.xml" lowUpperBound="35" highLowerBound="70"/>
+    <log type="coverage-text" target="php://stdout" showOnlySummary="true" showUncoveredFiles="false"/>
+  </logging>
+  <php>
+    <env name="APP_ENV" value="testing"/>
+    <env name="TELESCOPE_ENABLED" value="false"/>
+    <env name="BCRYPT_ROUNDS" value="4"/>
+    <env name="CACHE_DRIVER" value="array"/>
+    <env name="SESSION_DRIVER" value="array"/>
+    <env name="QUEUE_DRIVER" value="sync"/>
+    <env name="MAIL_DRIVER" value="array"/>
+    <env name="DB_CONNECTION" value="sqlite"/>
```

Comment:
use `testing` to use `:memory:` by default. Or was this connection introduced after Laravel 5.3? :/